charts/karpenter/values.yaml

```diff
@@ -121,7 +124,14 @@ controller:
   # - name: AWS_REGION
   #   value: eu-west-1
   envFrom: []
-  # -- Resources for the controller pod.
+  securityContext:
+    # -- AppArmor profile for the controller container.
+    appArmorProfile: {}
+    # -- SELinux options for the controller container.
+    seLinuxOptions: {}
+    # -- Seccomp profile for the controller container.
+    seccompProfile: {}
```

Comment:
@stevehipwell Do we want to change the SecComp profile from what it was where it was set to RuntimeDefault?

Reply:
This is will inherit the parent value which we are setting (line 57) [docs](https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.26/#securitycontext-v1-core).